core/tests/puppeteer-acceptance-tests/puppeteer-testing-utilities/test-constants.js

```diff
@@ -20,13 +20,13 @@ const path = require('path');
 
 let testConstants = {
   URLs: {
-    home: 'http://localhost:8181/',
-    BlogDashboard: 'http://localhost:8181/blog-dashboard',
-    BlogAdmin: 'http://localhost:8181/blog-admin',
-    CreatorDashboard: 'http://localhost:8181/creator-dashboard',
-    AdminPage: 'http://localhost:8181/admin',
-    RolesEditorTab: 'http://localhost:8181/admin#/roles',
-    logout: 'http://localhost:8181/logout'
+    home: 'http://localhost:9001/',
```

Comment:
Why did using 8181 not cause an error previously? Are these constants used at all?

Reply:
it defines the port utilized by `google_app_engine` so before i was using `8181` for the sake for that it won't confuse people. Suppose a new person is writing a new test, he would open a dev server and try to find the necessary solution, but the devserver port is always reserved at `8181`. But now since we have moved the `google_app_engine` port to 9001 only for testing it may or can cause some confusion among contributors. But i think it's alright since we have constants defined, so people won't have that much confusion regarding this.

Reply:
@ashish-patwal But what process uses this? Can the whole dict be removed if it's not actually being used?

Reply:
@seanlip it's just dict of constants mentioning the addresses of various pages like home, dashboard etc. It is done to use a more generic value instead of manually typing the url address like `localhost:9001/home`

Reply:
Thanks, I get it now. I left some suggestions for name changes because otherwise if you refer to 9001 as the standard GAE port and put that in common.py, that's going to confuse someone sooner or later. PTAL.